### PR TITLE
Added unimodule support

### PR DESCRIPTION
**Pull request by EvanBacon**
- Adds support for the Expo SDK
- Updated SSR formatting
- Corrected module resolution with `resolvableExtensions`
- Updated babel preset to use platform based tree-shaking and decorator support
- Reduced overall config required for starting a React Native project in Gatsby
- Removed kittens support - I think we shouldn't resolve `.android.ts` files in web.

### CODE REVIEW

File: examples/uikits/src/pages/expo.js [hunk 1]
```diff
@@ -1,39 +1,189 @@
...
+      </Text>
+    </ScrollView>
+  )
+}
+
+function FontExample() {
+  const [loaded, setLoaded] = useState(false)
+
+  useEffect(() => {
+    ;(async () => {
```
Comment (slorber) [inline examples/uikits/src/pages/expo.js]: ; is a typo? or is this useful for some reason?

Comment (EvanBacon) [reply examples/uikits/src/pages/expo.js]: I think it's a new ESLint bug going around, will remove.

**Comment by EvanBacon** [comment]
I'll work on adding UI Kittens back without shimming to Android

File: README.md [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
 [![Build Status](https://travis-ci.com/slorber/gatsby-plugin-react-native-web.svg?branch=master)](https://travis-ci.com/slorber/gatsby-plugin-react-native-web)
 
-
 Share components between your React Native mobile app and your Gatsby static website.
 
-# Install
+# Setup
+
+1. Install the plugin: `yarn add gatsby-plugin-react-native-web` or `npm install --save gatsby-plugin-react-native-web`
+2. Create a `gatsby-config.js` add use the plugin:
```
Comment (EvanBacon) [inline README.md]:
```suggestion
2. Create a `gatsby-config.js` and use the plugin:
```

**Comment by EvanBacon** [comment]
Ok, this should be good to merge now on my side.

**Comment by rickiesmooth** [comment]
tested this and it works great, thank you!

**Comment by slorber** [comment]
great!

@EvanBacon you should have permission to merge yourself and publish if you think you are done. Tell me if it's not the case.

The PR looks good to me

**Comment by slorber** [comment]
Thanks @EvanBacon 

Unless you want to change something, I'm going to merge it this week, and try to write a blog post as well on why you might want to consider using Gatsby for RNW in the first place (as I think people might not figure this out themselves)

**Comment by EvanBacon** [comment]
@slorber sorry I was out of town. I don't have write access to this repo so I get:
```
This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.
```

**Comment by slorber** [comment]
np, nice talk btw ;)

I've sent you an invite but I think you didn't accept it
You'll likely find it here: https://github.com/slorber/gatsby-plugin-react-native-web/invitations

**Comment by slorber** [comment]
Great. Will write a blog post this weekend (including some expo in MDX) 
Thanks

**Comment by slorber** [comment]
Hi @EvanBacon 

I've tried to see why the UIKits example didn't build on Netlify and it seems I can't either build it locally.

For weird reasons, I get transpilation errors. It works with yarn start, but it fails with yarn build.
Were you successful running yarn build on this example UIKit? 

I get errors such as:

image

The Netlify log is: https://gist.github.com/slorber/444338633cd7e0eb9105423436099063

Any idea what could be the reason? Didn't look yet into the webpack plugin but maybe its config is not suited for node SSR? Don't really understand why but any idea what could be the reason some files transpile correctly with yarn start and not yarn build?

Noticed a new "babel.config.js" at the root, is this actually used?

**Comment by slorber** [comment]
Hi @EvanBacon 
I'm still trying to investigate the issues but not sure what's happening.

It looks like in dev mode Gatsby is able to transpile the node modules libraries, while it's not in prod mode (the input webpack config is not the same in dev/prod).

gatsby build works for regular RNW, but as soon as I add a lib (like ui kittens or paper) it stops working



here's the before/after webpack config
https://gist.github.com/slorber/5a34f2b85f13d87c19a2691a7060e5af

will try to compare it to the dev config soon, but if you have any idea what's going on i'd be interested to know ;)

**Pull request merged**